Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -16,7 +16,7 @@ public struct IndexModel: Encodable {
 
     /// Gets the default name for this index.
     internal var defaultName: String {
-        return String(cString: mongoc_collection_keys_to_index_string(self.keys.data))
```

Comment:
libmongoc's keys to index name function only supports `Int32`'s for the directions, so I had to rewrite it in Swift.

Reply:
Filed [CDRIVER-3084](https://jira.mongodb.org/browse/CDRIVER-3804)